pkg/global/BUILD.bazel

```diff
@@ -19,5 +19,7 @@ go_library(
         "@io_opencensus_go_contrib_exporter_jaeger//:jaeger",
         "@io_opencensus_go_contrib_exporter_prometheus//:prometheus",
         "@io_opencensus_go_contrib_exporter_stackdriver//:stackdriver",
+        "@org_golang_google_grpc//codes",
```

Comment:
I just saw you pushed an additional commit to move these dependencies to the right place in the BUILD file. Life pro tip: you don't need to modify BUILD files by hand for changes like this. Just run:
```
bazel run //:gazelle
```
And it will add the missing dependencies for you!

Reply:
Nice. For these sorts of commands, probably worth documenting them in the README? I am more often in projects with the regular Go toolchain, so I always end up forgetting Bazel invocations like this and have to relearn every time ...